dimagi/commcare-hq
# Loading Stripe.js directly from Stripe

## Technical Summary

Ticket: https://dimagi.atlassian.net/browse/SAAS-14652

This ticket delete the stripe.js file we put in our repo and load stripe.js directly from stripe

## Safety Assurance

### Safety story

I tested locally, there are no error or warnings in the console anymore.
I also put it on staging, built successfully, also no errors and warnings in the console.
I test several views that use stripe.js, still functional.

### Automated test coverage



### QA Plan


QA Ticket: https://dimagi.atlassian.net/browse/QA-6702
QA should perform all operations in HQ that use that stripe JS library. Mainly billing stuff: setup payment, change billing info, update billing contact ...

### Rollback instructions




### Labels & Review
